lib/v2/screens/app/interactor/viewmodels/app_event.dart

```diff
@@ -72,3 +73,12 @@ class OnApproveGuardianRecoveryDeepLink extends AppEvent {
   @override
   String toString() => 'OnApproveGuardianRecoveryDeepLink';
 }
+
+class OnSigningRequest extends AppEvent {
+  final ScanQrCodeResultData esr;
+
+  const OnSigningRequest(this.esr);
+
+  @override
+  String toString() => 'OnSigningRequest';
```

Comment:
is `esr` sensitive data? 
If not, should we print is as part of the toString?

Reply:
it's not sensitive data, no

Reply:
I am not sure what and how we're using toString - is this just for our private debugging, or any plans to put it in production logs too?

Personally when debugging I turn off state printouts because it's too much information - signal/noise goes bad. 

I like targeted printouts.

Reply:
Our base state mapper prints events. 
We have a trace of events that happen leading to a certain place. 

Thats what the toStrings are used for.